src/Cli/dotnet/commands/dotnet-workload/install/IInstaller.cs

```diff
@@ -12,6 +13,7 @@ namespace Microsoft.DotNet.Workloads.Workload.Install
     internal interface IInstaller : IWorkloadManifestInstaller
     {
         int ExitCode { get; }
+        SdkFeatureBand SdkFeatureBand { get; }
```

Comment:
I don't think we need to add this property.  It looks like the only place it's used is in WorkloadHistoryRecorder, and there we can use the existing `IWorkloadResolver.GetSdkFeatureBand` method instead.